src/database/migrations/20210107065115-create-location.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("Locations", {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      city: {
+        type: Sequelize.STRING,
+        defaultValue: true,
```

Comment:
default value `true` on `city` and `country` fields doesn't sound like a better option!